src/worker/shutdown.sh

```diff
@@ -3,9 +3,9 @@
 arch=$1
 
 kill_exporters() {
-    pkill -f "${1}_exporter.py*" && sleep 1
```

Comment:
what was the motivation behind the sleep when it was initially added?

Reply:
This was an artifact of when it was being called from the Ansible script vs the shutdown.sh script. In any case I did test without the sleep commands and it functions correctly. Do you think its worth asking MSR?

Reply:
yes, please let's check with MSR to double confirm.

Reply:
I reached out to Yifan via email. Waiting for a response.

Reply:
@jithinjosepkl, just got word from Yifan. The sleep commands where part of the original deployment scripts where the shutdown of each exporter was subsequently followed by the startup of the exporter. In the original script sleeps for a second to allow sufficient time for the process to die and the ports to free up before startup.

How the scripts are structured today the startup of the exporters are done later and not right after shutdown. Additionally, there is a 3 second sleep at the end of this script that would allow for sufficient time before exporter start up.